msfdb_helpers/db_interface.rb

```diff
@@ -0,0 +1,35 @@
+class DbInterface
+
+  def init_db
```

Comment:
Should we nuke the `_db` suffix now that they're named already somewhat namespaced under a `db` class?

Reply:
I actually started without the `_db` suffix and then added it in later :upside_down_face: 
I'm down to remove it